.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: MindMeld
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6]
+
+    steps:
+    - uses: actions/checkout@v2
+  
+    - uses: nyaruka/elasticsearch-action@v1
+      with:
+        elastic version: '6.8.5'
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        java-version: '9.0.4' # The JDK version to make available on the path.
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64 # (x64 or x86) - defaults to x64
+    
+    - name: Install dependencies
+      run: |
+        ./scripts/travis_install.sh
```

Comment:
why install travis ci when I assume github actions replicates it?

Reply:
This is a poorly named script but basically it installs all of our dependency.